samples/vSphereAutoRestartManager.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+import sys
+import atexit
+from pyVmomi import vim
+from pyVim.connect import Disconnect, SmartConnect
+sys.dont_write_bytecode = True
+
+
+# you need to pip install pyCLI
+from cli.log import LoggingApp
```

Comment:
If you're adding a new dep you should add it to the requirements.txt file as well